packages/admin-stories/src/admin-rte/RteTheming.tsx

```diff
@@ -0,0 +1,29 @@
+import { makeRteApi, Rte } from "@comet/admin-rte";
+import { Box, Card, CardContent } from "@material-ui/core";
+import { select } from "@storybook/addon-knobs";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+
+const [useRteApi] = makeRteApi();
+
+function Story() {
+    const { editorState, setEditorState } = useRteApi();
+
+    return (
+        <>
+            <Box marginBottom={4}>
+                <Card variant="outlined">
+                    <CardContent>
+                        <Rte
+                            value={editorState}
+                            onChange={setEditorState}
+                            theme={{ editor: { minHeight: select("minHeight", [0, 100, 200, 300, 1000], 0) } }}
```

Comment:
I think changing the minHeight is a common property to change, so I propose to expose it like `<Rte minHeight={100} />`.